src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,24 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    public static final String CAN_T_BUILD_SESSION_FACTORY_MSG = "Can't build session factory ";
+    private static final SessionFactory session = getInitSessionFactory();
+
+    public HibernateUtil() {
+    }
+
+    private static SessionFactory getInitSessionFactory() {
+        try {
+            return new Configuration().configure().buildSessionFactory();
+        } catch (Exception e) {
+            throw new RuntimeException(CAN_T_BUILD_SESSION_FACTORY_MSG, e);
```

Comment:
The `RuntimeException` should include the original exception as a cause to preserve the stack trace. Modify the exception message to `throw new RuntimeException(CAN_T_BUILD_SESSION_FACTORY_MSG, e);` to include the original exception.